CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.9.0 (28 July, 2021)
+
+### Enhancements
+
+ * Drop dependency on vscode-commons by updating `@redhat-developer/vscode-redhat-telemetry` to 0.2.0. See [#365](https://github.com/redhat-developer/vscode-quarkus/issues/365).
+ * Add a command to deploy a Quarkus app to OpenShift using OpenShift Connector. See [#313](https://github.com/redhat-developer/vscode-quarkus/issues/313).
+ * Emit telemetry events for command executions. See [#344](https://github.com/redhat-developer/vscode-quarkus/issues/344).
+ * Add a command to build a binary of a Quarkus app. See [#341](https://github.com/redhat-developer/vscode-quarkus/pull/341).
+ * RESTEasy JAX-RS is no longer added by default in the project generator wizard. See [#322](https://github.com/redhat-developer/vscode-quarkus/issues/322).
+ * Add a step to the project generator to specify if example code should be generated. See [#301](https://github.com/redhat-developer/vscode-quarkus/issues/301).
+
+### Bug Fixes
+
+ * Starting debugger in a Quarkus project that contains a `node_modules/` fails. See [#339](https://github.com/redhat-developer/vscode-quarkus/issues/339).
```

Comment:
Change to "Fixed debugger start-up failure in a Quarkus project that contains a `node_modules/` folder. See [#339](https://github.com/redhat-developer/vscode-quarkus/issues/339)."